packages/shared/lib/services/connection.service.ts

```diff
@@ -811,7 +811,7 @@ class ConnectionService {
             }
 
             case 'OAUTH2_CC': {
-                if (!rawCreds['access_token'] && !rawCreds['data']['token']) {
+                if (!rawCreds['access_token'] && !(rawCreds['data'] && rawCreds['data']['token']) && !rawCreds['jwt']) {
```

Comment:
Not necessarily, since `JWT` is not universally present in all `OAuth2_CC` instances. The current structure ensures flexibility to handle different scenarios in `rawCreds`

Reply:
ah yes I see, all good 👍🏻